extensions/visualizations/visualizations.js

```diff
@@ -62,6 +62,45 @@ oppia.directive('oppiaVisualizationBarChart', [function() {
   };
 }]);
 
+// TODO(brianrodri): These controllers may benefit from being reused
```

Comment:
Hm, this is ... interesting. I have some questions:

- Why are we declaring these globally? We shouldn't declare anything globally, it ends up sitting on the `window` object -- which is not great. 
- All these return what are basically constants. Why don't we do this declaratively, with a static object?
- Why are we doing this at all? It is better _not_ to use ng-bind-html where possible, since it's a pretty heavy hammer. Also this seems to introduce a bunch of indirection which I'm not sure is necessary.